### ISSUE TITLE
With native titlebar, cannot navigate between menus with left/right arrow keys

### ISSUE THREAD

**Issue by JohnKlenk** [microsoft/vscode#116876]
Issue Type: <b>Bug</b>

With the native titlebar enabled, you cannot navigate between menus with the left/right arrow keys.  This *IS* an accessibility issue.  I avoid the mouse as much as possible because it is painful to use. Being able to quickly and easily switch between pulldown menus with the left/right arrow keys is very useful to me, to quickly find which menu has what I'm looking for with the least possible repetitive motion.

VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:04.245Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz (8 x 2712)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|23.79GB (16.42GB free)|
|Process Argv|--crash-reporter-id 42431fa7-02b8-4885-aa66-fb6b0034b665|
|Screen Reader|no|
|VM|29%|
</details><details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|11.2.1
vscode-pull-request-github|Git|0.23.1
cpptools|ms-|1.2.1

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt617cf:30253240
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30256636
pythonvsdeb440:30248342
pythonvsded773:30248341
vstes627:30244334
pythonvspyt600cf:30251589

```

</details>

**Comment by sbatten** [comment]
the custom menubar was made to address accessibility issues that wouldn't be prioritized in electron. assigning this issue to @deepak1556 to assess.

**Comment by JohnKlenk** [comment]
Any chance of improving the custom titlebar so that you can easily see when VS Code has focus?  To me that seems like a basic requirement of any app.  Keyboard users rely on knowing where their input will go...

**Comment by sbatten** [comment]
@JohnKlenk we provide the following theme keys that help differentiate active windows.

```json
"titleBar.inactiveBackground": "#999999",
"titleBar.inactiveForeground": "#ffffff",
"titleBar.activeForeground": "#999999",
"titleBar.activeBackground": "#ffffff"
```


These are not suggested colors but examples. If our default themes are not clear enough, maybe @misolori can help here.

**Comment by JohnKlenk** [comment]
Thank you.  I used the following in my settings.json to get as close as I could to the gold standard of the Windows Classic titlebar, which is the most functional and best looking of any that I've ever seen on any OS:
`    "workbench.colorCustomizations": {`
`        "window.activeBorder": "#000000",`
`        "titleBar.inactiveBackground": "#898989",`
`        "titleBar.inactiveForeground": "#C7D0C8",`
`        "titleBar.activeForeground": "#ffffff",`
`        "titleBar.activeBackground": "#1D397B"`
`        },`

I still hope the various accessibility issues with the native titlebar can be fixed since to me that has the best visual appearance, but for now I'll use the above changes.

**Comment by deepak1556** [comment]
Closing as not actionable in the runtime.
